hazelcast/config.py

```diff
@@ -559,6 +584,11 @@ class ClientProperties(object):
     Pause time between each retry cycle of an invocation in milliseconds.
     """
 
+    HAZELCAST_CLOUD_DISCOVERY_TOKEN = ClientProperty("hazelcast.client.cloud.discovery.token", "")
+    """
+    Token to use when discovering cluster via hazelcast.cloud
```

Comment:
`Hazelcast.cloud` you can probably do replacement with hazelcast.cloud -> Hazelcast.cloud.